### ISSUE TITLE
Bug when logged in to Wordpress

### ISSUE THREAD

**Issue by operapreneur** [davist11/jQuery-Stickem#41]
Thanks for your plugin. I can't get to the bottom of a bug. I have your plugin working correctly in a custom Wordpress theme but when I'm logged in the 'stickit' class is added before the window scrolls to the container. I thought about using the offset value because of the fixed menu that is added by Wordpress but the class is added when the container scrolls passed the middle of the browser window - a dimension much larger than the fixed menu. When I try to inspect the element the problem fixes itself.

Any ideas on how to get to the bottom of this? Unfortunately I can't paste any public links. Thanks!

**Comment by davist11** [comment]
Is this because you get an admin bar once you are logged in?

**Comment by operapreneur** [comment]
I thought it was but I believe I fixed it.

I gave the element below my stickem container (an HTML5 footer with a background-image, defined height, and a negative top margin) a position of relative. I'm not exactly sure why this would only happen when I was logged in. I'm guessing the container height was being thrown off by the admin bar and ill-defined position.

Thanks for the very useful plugin and quick reply! Cheers.
